package.json

```diff
@@ -36,6 +36,9 @@
     "redux-thunk": "2.3.0",
     "semver": "7.3.2",
     "style-loader": "1.2.1",
+    "vega": "5.13.0",
```

Comment:
Are both `vega` and `vega-lite` necessary here? I noticed the bundle size jumps to ~12MB after adding these.

Reply:
Yes, it looks like `vega-lite` does a bunch of imports from `vega`.  However, tree shaking will get rid of what we don't use, so if you set `mode: 'production'` instead of `mode: 'development'` in `webpack.config.js`, the bundle size goes down to 2.8MB.